### PR TITLE
[Fix] Issue #2306

### PR DESCRIPTION
**Pull request by camposandro**
## Description

This pull request aims to enable the user to choose whether the update dialog pop-ups should or should not show up, while in use of the application.

issue1

## Issue fixed

[#2306](https://github.com/BoostIO/Boostnote/issues/2306)

## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :radio_button: Improvement (Change that improves the code. Maybe performance or development improvement)
- :radio_button: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist

- :radio_button: My code follows [the project code style](docs/code_style.md)
- :white_circle: I have written test for my code and it has been tested
- :radio_button: All existing tests have been passed
- :radio_button: I have attached a screenshot/video to visualize my change if possible

### CODE REVIEW

**Comment by Margarida-Silva** [comment]
We would really appreciate to know whether this PR is ready to be accepted, hence we are working on this open source project in the context of a university course and it would be beneficial to discuss this PR, or even see it being approved.

File: browser/main/UpdatePopup/UpdatePopup.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import i18n from 'browser/lib/i18n'
+import ConfigManager from 'browser/main/lib/ConfigManager'
+import store from 'browser/main/store'
+
+const electron = require('electron')
+const { remote, ipcRenderer } = electron
+const { dialog } = remote
+
+function updateApp () {
```
Comment (Rokt33r) [inline browser/main/UpdatePopup/UpdatePopup.js]: Please export function directly.

```
export function updateApp() {
```

File: browser/main/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { Router, Route, IndexRoute, IndexRedirect, hashHistory } from 'react-rou
 import { syncHistoryWithStore } from 'react-router-redux'
 require('./lib/ipcClient')
 require('../lib/customMeta')
-import i18n from 'browser/lib/i18n'
+import UpdatePopup from './UpdatePopup/UpdatePopup'
```
Comment (Rokt33r) [inline browser/main/index.js]: Fix this too.

`import { updateApp } from 'lib/updateApp`

File: browser/main/index.js [hunk 3]
```diff
@@ -133,7 +119,7 @@ ReactDOM.render((
     notify('Update ready!', {
       body: 'New Boostnote is ready to be installed.'
     })
-    updateApp()
+    UpdatePopup.updateApp()
```
Comment (Rokt33r) [inline browser/main/index.js]: Rollback to `updateApp()`

**Comment by Rokt33r** [comment]
@Margarida-Silva Sorry for our late response.
@camposandro Please resolve my change request. I'll merge this pr ASAP. 🚀

**Comment by camposandro** [comment]
@Rokt33r Thanks for your review, I believe I have just corrected everything you mentioned!

**Comment by Rokt33r** [comment]
Sadly, I found this pr is irrelevant to #2306. :sob::sob: It is talking about the notification not the dialog box. 

@camposandro @Margarida-Silva I think I have to close this pr and make another pr to solve this problem.

I've added new spec in https://github.com/BoostIO/Boostnote/issues/2827. If you could afford, please try this one.

**Pull request closed without merging**